control-plane/connect-inject/controllers/pod/pod_controller_ent_test.go

```diff
@@ -194,7 +194,7 @@ func TestReconcileCreatePodWithMirrorNamespaces(t *testing.T) {
 			expectedWorkload:           createWorkload(),
 			expectedHealthStatus:       createPassingHealthStatus(),
 			expectedProxyConfiguration: createProxyConfiguration(testPodName, pbmesh.ProxyMode_PROXY_MODE_DEFAULT),
-			expectedUpstreams:          createUpstreams(),
```

Comment:
should be `existingDestinations:`

Reply:
Yep, looks like I forgot the ent test file!